package.json

```diff
@@ -26,7 +26,11 @@
     "@fullcalendar/react": "^6.1.11",
     "@fullcalendar/timegrid": "^6.1.11",
     "@rails/ujs": "^7.1.3",
+    "@stripe/react-stripe-js": "^2.7.0",
+    "@stripe/stripe-js": "^3.3.0",
     "@tanstack/react-query": "^5.28.9",
+    "@thewca/wca-components": "^0.6.0",
```

Comment:
I would prefer not to include `wca-components` at this point in time. See my lengthier comment above. Is there any use case in this PR other than `UiIcon` and `CubingIcon`?